tests/integration-node/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "miden-integration-node-tests"
+version.workspace = true
+rust-version.workspace = true
+authors.workspace = true
+repository.workspace = true
+categories.workspace = true
+keywords.workspace = true
+license.workspace = true
+readme.workspace = true
+edition.workspace = true
+publish = false
+
+[dependencies]
+anyhow.workspace = true
+cargo-miden.workspace = true
+fs2 = "0.4"
+miden-client = { version = "0.10", features = [
+    "tonic",
+    "sqlite",
+] }
+miden-core.workspace = true
+miden-mast-package.workspace = true
+miden-objects = { workspace = true, features = ["std"] }
+midenc-debug.workspace = true
+midenc-frontend-wasm.workspace = true
+midenc-session.workspace = true
+proptest.workspace = true
+rand = "0.9"
+temp-dir = "0.1"
+tokio.workspace = true
+uuid = { version = "1.10", features = ["v4"] }
+
+# For accessing the test builder from the main integration tests
+miden-integration-tests = { path = "../integration" }
```

Comment:
Maybe we should move shared test code like that into a `midenc-testing` crate, and then share it between `miden-integration-tests` (which we should probably rename to `midenc-integration-tests) and this crate?

Having this crate depend on `miden-integration-tests` sort of defeats the point of splitting it out into its own crate IMO.

Reply:
Good point! I agree.